src/rdiff_backup/fs_abilities.py

```diff
@@ -224,7 +224,7 @@ def set_extended_filenames(self, subdir):
 		ord_rp.delete()
 
 		# Try a UTF-8 encoded character, it's a path as string FIXME?
-		extended_filename = 'uni' + chr(225) + chr(132) + chr(137)
+		extended_filename = ('uni' + chr(225) + chr(132) + chr(137)).encode('utf-8')
```

Comment:
The utf-8 encoding i think makes sense because `chr(225) + chr(132) + chr(137)` ist the utf-8 byte representation of `á`. `b'uni' + bytes(bytearray((225,132,137)))` gives the same result as `('uni' + chr(225) + chr(132) + chr(137)).encode('utf-8')` or `'uniá'.encode('utf-8')`. later i would prefer to have `'uniá'.encode('utf-8')` here, but i was not sure if the project has a source encoding of utf-8 specified. it seems at leas fs_abilities has not, we propably should specify the encoding after https://github.com/rdiff-backup/rdiff-backup/pull/111 is merged.